SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/collaborations.md

To retrieve a list of the pending collaborations requiring the user to accept or reject them, call
[`client.collaborations.listPendingForEnterprise(offset:limit:fields:)`][get-pending-collaborations].

> The method returns an iterator in the completion that can be used to get pending collaborations.

Reviewer: ```suggestion
The method returns an iterator in the completion, which is used to get pending collaborations.
```
